app/controllers/pupils/schools_controller.rb

```diff
@@ -1,4 +1,6 @@
 class Pupils::SchoolsController < SchoolsController
+  include ActionView::Helpers::NumberHelper
```

Comment:
It needs this because we're in a controller right?

Reply:
Yes, only available in views by default.

Reply:
If we did more work on the equivalences it might be worth moving the text creation somewhere else.